index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] At the moment, the styling for the page is not visible. I think this is because you are linking the stylesheet with `style.css` but your file is named `Style.css`. I suggest renaming the file to `style.css`